Conceptpower+Spring/src/main/webapp/WEB-INF/views/layouts/footer.html

```diff
@@ -0,0 +1,7 @@
+<div class="footerBox">
+   <div class="footer">
+      <h5>Version: <span th:text="${#strings.defaultString(buildNumber,'')}"></span> 
+      <span th:if="${pullrequest != null}" >, Pull Request: <span th:text="${#strings.defaultString(pullrequest,'')}"></span></span></h5>
```

Comment:
`${pullrequest}` needs to stay like that so that Maven replaces it with the proper value. Here, it is not soemthing that's coming from the model/controller but that is replaced by maven and is then a static value in the templates. The alternative and long term probably better way to do this, is to write a property in a config file and get the value from there.